frontend/components/DisplayButtons.tsx

```diff
@@ -72,6 +73,14 @@ const DisplayButtons = ({
       <Icon name="shuffle" alt="shuffle club order" style={iconStyles} />
       &nbsp;&nbsp; Shuffle
     </button>
+    <button
+      onClick={switchSort}
+      style={{ ...buttonStyles, color: DARK_GRAY, fontWeight: 600 }}
+      className="button is-small"
+    >
+      <Icon name="filter" alt="shuffle club order" style={iconStyles} />
```

Comment:
This could potentially be a drop down with different options, like:
- Recommended
- Bookmarks
- Name
- etc